shuffle/cli/src/test.rs

```diff
@@ -146,20 +143,26 @@ pub fn run_move_unit_tests(project_path: &Path) -> Result<UnitTestResult> {
         ..UnitTestingConfig::default_with_bound(None)
     };
 
+    // Default publishing address to a placeholder address for Move unit tests,
+    // which do not run against a Node, but solely in the Move VM.
+    let publishing_address = AccountAddress::from_hex_literal(shared::PLACEHOLDER_ADDRESS)?;
```

Comment:
chatted offline regarding placeholder address use, potentially unifying it with account generation used in other tests, if not we can keep this as is

Reply:
Found out it's because shuffle test unit doesn't take a --network parameter atm, and the way our code path works now, all account creation connects to a network to get the correct sequence_number. Refactor happening post hackathon